src/Workspaces/Core/Portable/SymbolId/SymbolKey.BodyLevelSymbolKey.cs

```diff
@@ -63,6 +63,10 @@ public override SymbolKeyResolution Resolve(Compilation compilation, bool ignore
             private static IEnumerable<ValueTuple<TSymbol, int>> EnumerateSymbols(Compilation compilation, ISymbol containingSymbol, string name, CancellationToken cancellationToken)
             {
                 int ordinal = 0;
+                if (compilation == null)
```

Comment:
I think we should track that down, because clearly we don't understand this code enough to make this change.